### PR TITLE
Added persistance blockIndex

### PR DESCRIPTION
**Pull request by niklabh**
Closes: https://github.com/paritytech/Nomidot/issues/184
- added blockIndex in blockIndex table
- added IDENTIFIER so multiple runs can be done in parallel

### CODE REVIEW

File: back/node_watcher/src/util/blockIndex.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { BlockIndexCreateInput, prisma } from '../generated/prisma-client'
+
+export function createBlockIndex(identifier: string, startFrom: number, index: number) {
```
Comment (Tbaut) [inline back/node_watcher/src/util/blockIndex.ts]: I guess all these functions here could be used directly.
There is a stray `a` function and `updateBlockIndex` that is empty.

Comment (niklabh) [reply back/node_watcher/src/util/blockIndex.ts]: removed

**Comment by niklabh** [comment]
Unrelated lint is failing

**Comment by yjkimjunior** [comment]
could you just run `yarn lint --fix` locally and then push that?

**Comment by niklabh** [comment]
issue is in nomidot/back/server shouldn't it go in seperate PR.

**Comment by niklabh** [comment]
also lint command is not there in server or node_watcher

**Comment by yjkimjunior** [comment]
you can run it from either root or from /back, CI says it's a couple trailing commas so i think it's trivial enough to just fit into this PR

**Pull request merged**